linode/acceptance/provider_factories.go

```diff
@@ -26,3 +27,9 @@ var ProtoV5ProviderFactories = map[string]func() (tfprotov5.ProviderServer, erro
 		return muxServer.ProviderServer(), nil
 	},
 }
+
+var HttpExternalProviders = map[string]resource.ExternalProvider{
```

Comment:
Now this external provider is required for any tests that use Linode Cloud Firewall